### PR TITLE
first pass at clips page

### PR DESCRIPTION
**Pull request by gbinal**

### CODE REVIEW

File: pages/clips.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+permalink: clips/
+---
+<h1>New Clips</h1>
+
+{% for c in site.data.clips %}
```
Comment (mbland) [inline pages/clips.html]: For now, this'd need to be `site.data.public.clips`, unless you want to hoist `_data/public/clips.yml` up to `_data/clips.yml` in this here PR.

**Comment by mbland** [comment]
Also, do a `git pull origin master`, since the `.travis.yml` file doesn't appear in this branch yet.

**Comment by afeld** [comment]
Closes https://github.com/18F/hub/issues/38.

**Comment by gbinal** [comment]
Not having worked through @mbland's advice above, https://hub.18f.us/clips/ is currently blank.  

So tried correcting the data file's url with https://github.com/18F/hub/commit/2edc5611730dde77b0c38971e9e8b85eed677e1d.  

I'm operating through the browser though, so haven't been able to do `git pull origin master` as requested, though.

**Pull request merged**